### ISSUE TITLE
Remove "Char" from multi-key help output

### ISSUE THREAD

**Issue by lahwaacz** [kamiyaa/joshuto#34]
When I press e.g. `g`, joshuto prints a "help" output for the available multi-keys, such as:

```
   Char('/')    change_directory /
   Char('M')    change_directory /mnt
   Char('d')    change_directory /dev
   Char('e')    change_directory /etc
   Char('g')    cursor_move_home
   Char('h')    change_directory ~
   Char('m')    change_directory /media
   Char('n')    new_tab
   Char('o')    change_directory /opt
   Char('r')    change_directory /
   Char('s')    change_directory /srv
   Char('u')    change_directory /usr
   Char('v')    change_directory /var
```
Why does it say `Char('/')` and not just `/`?

Also, the output could be sorted better - notice that "M" and "m" are far from each other.

**Comment by kamiyaa** [comment]
should be fixed with 0cb2c16b4661762efbe3f8263ebb7837657cfb92
Also, `change_directory` has been changed to `cd` (as originally intended)

**Comment by lahwaacz** [comment]
Thanks!
